### ISSUE TITLE
attachable is not allowed

### ISSUE THREAD

**Issue by keithhaight** [mschnitzer/ark-survival-ascended-linux-container-image#87]
Running openSUSE Tumbleweed. Server shows in browser. Getting connection timeout. Working through instructions in ReadMe. After edititng docker-compose.yml and removing asa-server-1 and asa-server_asa-network; upon running            'docker-compose up -d' I recieve the following... 'validating /home/keith/asa-server/docker-compose.yml: services.asa-server-1.networks.asa-network Additional property **attachable is not allowed**'

Actually experienced the same issue with Debian. Did some research and thought it could have been related to my docker-compose version and SWARM support. Decided to reinstall with more up-to-date packages and openSUSE. Now running Docker Compose version 2.29.7

Thanks for all the effort that has gone into this project. Really appreciate all the work. I'm in way over my head with all this, but it's kinda fun trying to make it work. Thanks again!

**Comment by keithhaight** [comment]
Update: I have confirmed I can connect via the 'open' command from the in game console.

**Comment by mschnitzer** [comment]
I never used docker swarm before. How did your network section look like when you got that error?

**Comment by mschnitzer** [comment]
Is that still an error?

**Comment by keithhaight** [comment]
I'm sorry I can't help at this point, I abandoned the install. The network section was a standard install. The only change I made was from directions on the Github page for not being able to connect to the server. After making that change I started getting the error I reported.

My experience is very limited. Easily something maybe I was just doing, though I did follow the directions carefully.

I no longer have the install, so I don't have any further details... sorry.

Keith

Sent with [Proton Mail](https://proton.me/mail/home) secure email.
